src/metrics.cpp

```diff
@@ -225,12 +268,25 @@ int printStats(bool mining)
 
     if (IsInitialBlockDownload(Params())) {
         int netheight = currentHeadersHeight == -1 || currentHeadersTime == 0 ? 
-            0 : EstimateNetHeight(Params().GetConsensus(), currentHeadersHeight, currentHeadersTime);
+            0 : EstimateNetHeight(params, currentHeadersHeight, currentHeadersTime);
         int downloadPercent = height * 100 / netheight;
         std::cout << "     " << _("Downloading blocks") << " | " << height << " / ~" << netheight << " (" << downloadPercent << "%)" << std::endl;
     } else {
         std::cout << "           " << _("Block height") << " | " << height << std::endl;
     }
+
+    auto secondsLeft = SecondsLeftToNextEpoch(params, height);
+    std::string strUpgradeTime;
+    if (secondsLeft == boost::none) {
+        strUpgradeTime = "Unknown";
+    }
+    else {
+        auto nextHeight = NextActivationHeight(height, params).get_value_or(0);
```

Comment:
```suggestion
        auto nextHeight = NextActivationHeight(height, params).value();
```

Reply:
My mistake, plan was to get rid of all `get_value_or(0)`. Done now at https://github.com/zcash/zcash/pull/4181/commits/9582e258ba7b977cfb035aeadf07c80d7a8f7de7